node/forcedroid.js

```diff
@@ -300,7 +291,6 @@ function createNativeApp(config, showNextSteps) {
     // copy <Android Package>/libs/SalesforceSDK -> <App Folder>/forcedroid/libs/SalesforceSDK
     var salesforceSDKRelativePath = path.join('libs', 'SalesforceSDK');
     copyFromSDK(packageSdkRootDir, config.targetdir, salesforceSDKRelativePath);
```

Comment:
But to your point, we won't need this copy step - we can link it in ```build.gradle```. I'll do that in the next pull request.